16.5th Bug Crusade

zumbak04 opened the pull request:
Secret changelog:
- Fixed issue where WCCLS.500 had only 1/3 of text.
- Fixed issue where WCCLS.401 and WCCLS.500 had no localization if you were Kaldorei.

geffe1 commented:
![ck2_340](https://user-images.githubusercontent.com/31706375/64653713-7c8ded80-d427-11e9-925c-6422499f4033.png)
Normal priest one, feels ok. 
![ck2_339](https://user-images.githubusercontent.com/31706375/64653733-84e62880-d427-11e9-92b0-bfda7502a5ff.png)
The Night elf one however, needs to be rewritten to be about elune and all. So it all seems -OKAY-, as it works.. just doesn't feel right talking about the light instead of Elune for night elves.

geffe1 commented:
![ck2_342](https://user-images.githubusercontent.com/31706375/64655034-4e121180-d42b-11e9-83b8-5dab80b4fac9.png)
This one doesn't even appear for night elves though.

MemeWizard commented on localisation/00_zuwc_united_localisation.csv:
```diff
@@ -30,11 +30,11 @@ EVTDESC_WCCLS_100;You have completed your magical training by performing excepti
 EVTDESC_WCCLS_400;You have become a true warrior of the Holy Light, unbending to the forces of darkness that plague Azeroth. With great power comes great responsibility, as the forces of evil will always come to meet you as the Light's vanguard and beacon. May the weak and innocent of this world always find you as a guide and protector.;Vous �tes devenu un v�ritable chevalier de la Sainte Lumi�re ...
 EVTDESC_WCCLS_401;The powers of the divine do not always speak clearly, though when they do, they speak through their priests. These holy men and women are mouthpieces and weapons of the Holy Light. While not as armoured or as strong as paladins, priests channel pure divine magic to achieve their goals and protect the innocent.  It is a long line of study, and there will be many tests of faith, b ...
-EVTDESC_WCCLS_500;"The Light speaks to you very clearly now�; its powers of healing and preservation are a second language to you. You have left marks on those you have helped and protected with your divine words. The forces of darkness shy away from that which is a shining beacon of the Light: you. The pains and nightmares caused by evil find themselves utterly purged by your touch. May Azeroth  ...
+EVTDESC_WCCLS_500;The Light speaks to you very clearly now. Its powers of healing and preservation are a second language to you. You have left marks on those you have helped and protected with your divine words. The forces of darkness shy away from that which is a shining beacon of the Light: you. The pains and nightmares caused by evil find themselves utterly purged by your touch. May Azeroth se ...
```
French loc still have ; in the middle of sentence

zumbak04 replied:
Fixed!

zumbak04 commented:
> The Night elf one however, needs to be rewritten to be about elune and all. So it all seems -OKAY-, as it works.. just doesn't feel right talking about the light instead of Elune for night elves.

Added Elune flavour.

> This one doesn't even appear for night elves though.

Fixed and added Elune flavour.

MemeWizard commented:
> holy men

> implying male kaldorei priests

> male kaldorei can't become priests

ok

> not as strong as paladins

> implying kaldorei have paladins

> you can't become paladin as kaldorei

ok

<details>

![image](https://user-images.githubusercontent.com/46576860/64910441-2e2a6880-d71f-11e9-8639-d098f14341dc.png)

</details>

zumbak04 commented:
> > implying male kaldorei priests

> > implying kaldorei have paladins

Why Kaldorei? It speaks about priests in general.

MemeWizard commented:
>Why Kaldorei? It speaks about priests in general.

Then why event text mentions Elune?

zumbak04 commented:
> Then why event text mentions Elune?

Because night elves belive she gives them Light powers?

MemeWizard commented:
>Because night elves belive she gives them Light powers?

GOTO 1

Why do this event have general class description, but mention specific gods?

zumbak04 commented:
> GOTO 1
> 
> Why do this event have general class description, but mention specific gods?

Because priests, paladins everywhere are the same and using the same magic but believe different gods grant this power: Rezan, Elune, the Light, Naaru.

MemeWizard commented:
Where are my troll paladins then

zumbak04 commented:
> Where are my troll paladins then

In the same place where night elven paladins are
